### DISCUSSION TITLE
Imagick resizeImage(): What argument to use when not specifying maximum height?

### DISCUSSION THREAD

**Discussion by tmb-github** [ImageMagick/ImageMagick#5096] [Help]
I'm using imagick 3.7.0 compiled against ImageMagick 7.0.10-23 Q16 x86_64

In previous versions of imagick, I could resize an image with the width specified but the height unlimited using this command, where $width is a specific size but null is intended to mean unlimited for the height:

`resizeImage($width, null, imagick::FILTER_LANCZOS, 1);`

With the new imagick I've loaded to use PHP 8.1, the null is throwing an error on my server:

> Imagick::resizeImage(): Passing null to parameter #2 ($rows) of type int is deprecated

What is the proper argument to supply to the second parameter in this case?

Many thanks!

**Comment by dlemstra** [comment]
You should ask for IMagick help here instead: https://github.com/Imagick/imagick
